tests/Feature/WishlistTest.php

```diff
@@ -87,8 +87,25 @@
     Wishlist::factory()->for($wishlist->user)->create();
 
     $this->actingAs($wishlist->user);
-    $response = $this->delete(route('wishlists.destroy', $wishlist));
+    $response = $this->delete(route('wishlists.destroy', $wishlist), [
+        'original_name' => $wishlist->name,
+        'wishlist_name' => $wishlist->name,
+    ]);
 
     $response->assertRedirectToRoute('app');
     $this->assertSoftDeleted($wishlist);
 });
+
+test('users cannot delete their wishlist when enter wrong name', function () {
+    $wishlist = Wishlist::factory()->create();
+    Wishlist::factory()->for($wishlist->user)->create();
+
+    $this->actingAs($wishlist->user);
+    $response = $this->delete(route('wishlists.destroy', $wishlist), [
+        'original_name' => $wishlist->name,
+        'wishlist_name' => 'Incorrect name',
+    ]);
+
+    $response->assertSessionHasErrorsIn('wishlistDeletion', 'wishlist_name');
+    $this->assertNotNull($wishlist->fresh());
```

Comment:
```suggestion
    $this->assertModelExists($wishlist);
```
`assertModelExists` will check that the wishlist is still in the database. I think everything else looks good though.

Reply:
Works great!
![image](https://github.com/imacrayon/snowbodyknows/assets/11927587/fcbc16a5-19e8-498e-a606-91bd6965408b)